dll/ntdll/ldr/ldrapi.c

```diff
@@ -982,18 +984,19 @@ LdrVerifyImageMatchesChecksum(IN HANDLE FileHandle,
 
 NTSTATUS
 NTAPI
-LdrQueryProcessModuleInformationEx(IN ULONG ProcessId,
-                                   IN ULONG Reserved,
-                                   OUT PRTL_PROCESS_MODULES ModuleInformation,
-                                   IN ULONG Size,
-                                   OUT PULONG ReturnedSize OPTIONAL)
+LdrQueryProcessModuleInformationEx(
+    _In_opt_ ULONG ProcessId,
+    _Reserved_ ULONG Reserved,
+    _Out_writes_bytes_to_(Size, *ReturnedSize) PRTL_PROCESS_MODULES ModuleInformation,
+    _In_ ULONG Size,
+    _Out_opt_ PULONG ReturnedSize)
```

Comment:
https://github.com/reactos/reactos/blob/82241f85e4508ffe02bdb9538b570fc977069c59/dll/ntdll/ldr/ldrapi.c#L1098 seems odd:
it looks like `STATUS_SUCCESS` could be returned, though `*ReturnedSize` was not set, for example.

Reply:
Yes, it's odd (or incorrect), but I'm not planned to change the code in this PR for little fixes.